src/test/java/Search.java

```diff
@@ -0,0 +1,105 @@
+
+    String [] caseSenList = {"darks day", "darKs DAy"};
+    @Test(enabled = true, priority = 0)// dataProvider = "songTitles", dataProviderClass = BaseTest.class)
+    public void searchSongTitle() throws InterruptedException{
+
+//        login();
+        LoginPageSearch Login = new LoginPageSearch(driver);
+        SearchPageSearch searching = new SearchPageSearch(driver);
+        Login.login();
+
+        for (int i=0; i < list.length; i++) {
+            //System.out.println(list[i]);
+            searching.searchBox(list[i]);
+        }
+        //check for the results
+        resultFromSearch();
+    }
+    @Test(enabled = true,priority=1)
+    public void isSongSearchwithSentiveCase() {
+        LoginPageSearch Login = new LoginPageSearch(driver);
+        SearchPageSearch searching = new SearchPageSearch(driver);
+        Login.login();
+
+        for (int i=0; i < caseSenList.length; i++) {
+            searching.searchBox(caseSenList[i]);
+        }
+        songResultpage();
+        //negativeSearchResult();
+        //Assert.assertFalse(negativeSearchResult(),"should not shown");
+        Assert.assertEquals(negativeSearchResult(), false);
+
+    }
+     //expect to see result
+     public void resultFromSearch(){
+         //*[@id='searchExcerptsWrapper']//span[text()="Dark Days"]
+         WebElement songresult=driver.findElement(By.xpath("//*[@data-testid='song-excerpts']//button[@data-test='view-all-songs-btn']"));
+         songresult.click();
+
+         WebElement isThatCorrectSong = driver.findElement(By.xpath("//*[@id='songResultsWrapper']//td[@class='title']"));
+         Assert.assertEquals(isThatCorrectSong.getText(), "Dark Days");
+
+         WebElement isThatCorrectartist = driver.findElement(By.xpath("//*[@id='songResultsWrapper']//td[@class='artist']"));
+         Assert.assertEquals(isThatCorrectartist.getText(), "Grav");
+
+         WebElement isThatCorrectAlbum = driver.findElement(By.xpath("//*[@id='songResultsWrapper']//td[@class='album']"));
+         Assert.assertEquals(isThatCorrectAlbum.getText(), "Dark Days EP");
+        
+     }
+
+     public boolean negativeSearchResult() {
+
+         WebElement isThatCorrectSong = driver.findElement(By.xpath("//*[@id='songResultsWrapper']//td[@class='title']"));
+        return isThatCorrectSong.isDisplayed();
+         //return false;
+
+     }
+
+     public void songResultpage(){
+         WebElement songresult=driver.findElement(By.xpath("//*[@data-testid='song-excerpts']//button[@data-test='view-all-songs-btn']"));
+         songresult.click();
+
+     }
+     //not need yet
+//     public boolean isSongExisting(String songName) {
+//         WebElement allsongs = driver.findElement(By.xpath("//a[@href='#!/songs']"));
+//         allsongs.click();
+//        WebElement songText= driver.findElement(By.xpath("//*[@id='songsWrapper']//td[text()='"+ songName+"']"));
+//        return songText.isDisplayed();
+//     }
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.

![image](https://user-images.githubusercontent.com/104761482/209536974-3816c622-6918-46b0-80f6-75092515f3ea.png)

Reply:
I deleted all comments that not necessary.